flyve-mdm/web-mdm-dashboard
# Tests of 'components'

### Changes description


In this PR:

- Delete unnecessary 'EmptyList' component
- Delete unnecessary 'InputEmail' component
- Delete unnecessary 'ImageResponsive' component
- Change image of native notifications
- Validate the props of 'FilesUploadItemList'
- Add script to generate the coverage
- Delete unnecessary characters in the yarn scripts
- Create the following test:
  - EmptyMessage: 
    - create snapshot
  - ErrorValidation: 
    - create snapshot
  - HeaderBreadcrumb: 
    - create snapshot
    - create breadcrumb menu
  - InfoBox: 
    - create snapshot
  - IconItemList: 
    - create snapshot
    - should call componentDidMount
    - should get a local image
    - should mount a base64 image
  - TextArea: 
    - create snapshot
  - Select: 
    - create snapshot
  - DatePicker: 
    - create snapshot
    - should get the days of February in a leap year
    - should get the days of February
    - should get the months of a year
    - should get the years from 1980 to 2100
  - FilesUploadItemList: 
    - create snapshot
  - FilesUpload: 
    - create snapshot
  - LayoutListWithNavLinks: 
    - create snapshot

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Docs have been added/updated

### References



Closes #N/A
